src/MIDebugEngine/Engine.Impl/Breakpoints.cs

```diff
@@ -86,11 +86,15 @@ private static MIBreakpointState StringToBreakpointState(string addr)
 
         internal static async Task<BindResult> Bind(string functionName, DebuggedProcess process, string condition, AD7PendingBreakpoint pbreak)
         {
+            process.VerifyNotDebuggingCoreDump();
```

Comment:
Everywhere that I use the VerifyNotDebuggingCoreDump method, I've made sure all upstream callers catch the InvalidCoreDumpOperationException and return AD7_HRESULT.E_CRASHDUMP_UNSUPPORTED. Code on the VSCode side should be checking the hresult when calling these types of operations. In our case, we'll show a user-friendly error message stating something like "Unable to step next. This operation is not supported when debugging dump files". In the case of breakpoints, I was planning of having them show as set but unbound, so that the user doesn't have an expectation that they will be hit for some reason.